src/cmd/policy.go

```diff
@@ -21,6 +24,63 @@ type regoInput struct {
 	Files []string `json:"files"`
 }
 
+type gitlabRepoMetadata struct {
+	Name        string             `json:"name,omitempty"`
+	Description string             `json:"description,omitempty"`
+	Language    string             `json:"language,omitempty"`
+	Languages   map[string]float64 `json:"languages,omitempty"`
+}
+
+type githubRepoMetadata struct {
+	Name        string             `json:"name,omitempty"`
+	Description string             `json:"description,omitempty"`
+	Language    string             `json:"language,omitempty"`
+	Languages   map[string]float64 `json:"languages,omitempty"`
+}
+
+type genericStruct interface {
+	gitlabRepoMetadata | githubRepoMetadata | opslevel.Level
+}
+
+func structToValue[S genericStruct](s S) ast.Value {
+	marshalData, err := json.Marshal(s)
+	cobra.CheckErr(err)
+	reader := strings.NewReader(string(marshalData))
+	v, err := ast.ValueFromReader(reader)
+	cobra.CheckErr(err)
+	return v
+}
+
+func getMaxFromMap(m map[string]float64) string {
```

Comment:
Feels weird having to write this function here since I could see it being a common function but left it here for now.

Reply:
ya eventually we could pull this out to opslevel-common - but fine for now

Reply:
a better function name would be `getKeyFromMapByMaxValue`

its more verbose but it conveys exactly what the function does - self documenting